docs/source/pyro.optim.txt

```diff
@@ -1,23 +1,15 @@
-PyroOptim
----------
+Pyro Optimizers
+---------------
 
 .. automodule:: pyro.optim.optim
     :members:
     :undoc-members:
     :show-inheritance:
 
-ClippedAdam
```

Comment:
is there a reason not to combine these all under pyro optimizers? all the pytorch ones are wrapped like the custom ones.

Reply:
@neerajprad if you refactor optimizers as @jpchen  suggests, could you please check that docs links still work. it's a very delicate setup to link all the docs together

Reply:
>  if you refactor optimizers as @jpchen suggests, could you please check that docs links still work

Will do. I checked it with this PR, and will verify after removing the two classes again.